doc/tools/coqrst/repl/coqtop.py

```diff
@@ -54,17 +55,23 @@ def __init__(self, coqtop_bin=None, color=False, args=None) -> str:
             raise ValueError("coqtop binary not found: '{}'".format(self.coqtop_bin))
         self.args = (args or []) + ["-q"] + ["-color", "on"] * color
         self.coqtop = None
+        self.debugfile = None
 
     def __enter__(self):
         if self.coqtop:
             raise ValueError("This module isn't re-entrant")
         self.coqtop = pexpect.spawn(self.coqtop_bin, args=self.args, echo=False, encoding="utf-8")
         # Disable delays (http://pexpect.readthedocs.io/en/stable/commonissues.html?highlight=delaybeforesend)
         self.coqtop.delaybeforesend = 0
+        if not (os.getenv ("COQ_DEBUG_REFMAN") is None):
```

Comment:
```suggestion
        if os.getenv ("COQ_DEBUG_REFMAN"):
```

None is falsy.  Same thing in 3 other places.

Reply:
Alternatively, `is not None`; but not `not (… is None)`